### ISSUE TITLE
auto_update_tests.py fails since #6051

### ISSUE THREAD

**Issue by frank-emrich** [WebAssembly/binaryen#6076]
I'm getting the following error when running `scripts/auto_update_tests.py`:

```
Fatal: Failed opening '%S/asyncify-foo,bar-nl.txt'
[Failing module at line 6]
Traceback (most recent call last):
  < omitted >
subprocess.CalledProcessError: Command '/home/frank/projects/wasmfx/binaryen/scripts/foreach.py /home/frank/projects/wasmfx/binaryen/test/lit/passes/asyncify_pass-arg=asyncify-blacklist@foo,bar.wast /tmp/tmp8xngv6_g wasm-opt --asyncify --pass-arg=asyncify-blacklist@@%S/asyncify-foo,bar-nl.txt -S -o -' returned non-zero exit status 1.
Traceback (most recent call last):
  < omitted >
subprocess.CalledProcessError: Command '['/usr/bin/python3', '/home/frank/projects/wasmfx/binaryen/scripts/update_lit_checks.py', '--binaryen-bin=/home/frank/projects/wasmfx/binaryen/bin', '/home/frank/projects/wasmfx/binaryen/test/lit/**/*.wast', '/home/frank/projects/wasmfx/binaryen/test/lit/**/*.wat']' returned non-zero exit status 1
```

I bisected this error to being introduced by #6051, which does indeed look like it touched related code.

It may be useful to have a check similarly to that for `gen-s-parser.py` in the CI, ensuring that `auto_update_tests.py` succeeds and causes no changes (even though the latter would only be a kind of internal consistency check, I guess that the test suite passing should imply that auto_update_tests.py  should not cause changes)

**Comment by tlively** [comment]
This should be fixed by #6075, which I just merged. @frank-emrich, can you confirm?

**Comment by frank-emrich** [comment]
Yes, working now, thanks!
